CHANGELOG.md

```diff
@@ -2,6 +2,33 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.38.0](https://github.com/rudderlabs/rudder-transformer/compare/v1.37.0...v1.38.0) (2023-08-14)
```

Comment:
@aashishmalik remove the unwanted change log & check the reason for adding previous version change log history in current version

Reply:
updated changelog